### DISCUSSION TITLE
Would it make sense to link to the `next` version when using `to` links in the navbar?

### DISCUSSION THREAD

**Discussion by LukasGentele** [facebook/docusaurus#2959] [General] [answered]
I currently have multiple projects using docusaurus v2 and after starting `yarn start`, the browser opens and I always click on the navbar link "Docs" which in my case leads to `/docs/introduction`. However, if a project is versioned, I always catch myself being confused when I change something the markdown files, I see docusaurus compile the page but nothing happens in the browser. This is, of course, because I am in the latest stable version but my edits are within the `next` version.

Should links from the navbar automatically use the `next` version when in development mode? Are there other options, e.g. letting docusaurus open `/docs/next/introduction` instead of `/` when running `yarn start`?

**Comment by slorber** [comment, marked as answer]
Hi,

There are other issues related to this (https://github.com/facebook/docusaurus/issues/2302#issuecomment-642642112)

I don't think we should add different behaviors for dev and prod, that could be confusing.

However, I think we should be able to say which version is the "current one" (the one without any version path prefix) using configuration.


So you could do:

```
currentVersion: isDev ? "next" : undefined 
```

`undefined` means it would pick the last versioned version.

Does it make sense to you?

**Reply by LukasGentele** [reply]
So, I would need the isDev inside docusaurus.config.js. How can I get/determine if isDev == true? If there is a way to do that, I could use a `href` link instead of `to` and add the `next/` to the link.

**Reply by slorber** [reply]
In case you are not familiar with node ecosystem, you generally can do that with `process.env.NODE_ENV` that gives you "development" / "production"

So you could do:

const isDev = process.env.NODE_ENV === "development"

And use `isDev` where you need to.

**Reply by LukasGentele** [reply]
Sure, that makes sense. I'm familiar with this env var but I was thinking there was a docusaurus variable `isDev` or something that tells me if it's running as part of `yarn start` or as part of `yarn build` but of course the standard NODE_ENV will do. Didn't think of using that. Thanks! That is an easy fix then.

**Reply by LukasGentele** [reply]
In case anybody else looks for this, I ended up doing this in `docusaurus.config.js`:
```js
{
   href: __webpack_public_path__ + 'docs/' + (process.env.NODE_ENV == 'production' ? '' : 'next/') + 'introduction',
   label: 'Docs',
   position: 'left',
   target: '_self'
},
```
Of course, the `__webpack_public_path__ +` as well as the `+ 'introduction'` may be different in your case. And there may be a way to do this with `to` instead of `href` which I am not aware of.
